utils.py

```diff
@@ -77,7 +77,7 @@ def run_planning_demo(env, planner_name, outdir='/tmp', fps=3, verbose=False, se
     plan = run_planner(debug_info['domain_file'], debug_info['problem_file'], planner_name)
 
     actions = [parse_plan_step(s, env.domain.operators.values(), env.action_predicates,
-                debug_info["objects"], operators_as_actions=env.operators_as_actions) \
+                obs.objects, operators_as_actions=env.operators_as_actions) \
```

Comment:
Formatting seems a bit crazy here. Maybe something like this would be a bit clearer:
```python
actions = []
for s in plan:
    a = parse_plan_step(
        s,
        env.domain.operators.values(),
        env.action_predicates,
        obs.objects,
        env.operators_as_actions,
    )
    actions.append(a)
```

Reply:
fixed